protobuf/grpc_service.proto

```diff
@@ -1035,6 +1035,60 @@ message InferBatchStatistics
   StatisticDuration compute_output = 4;
 }
 
+//@@
+//@@.. cpp:var:: message InferResponseStat
+//@@
+//@@   Inference response statistics.
+//@@
+message InferResponseStat 
+{
+  //@@  .. cpp:var:: StatisticDuration compute_infer
+  //@@
+  //@@     The count and cumulative duration that generate the inference
+  //@@     results for a response.
+  //@@
+  StatisticDuration compute_infer = 1;
+
+  //@@  .. cpp:var:: StatisticDuration compute_output
+  //@@
+  //@@     The count and cumulative duration to extract the output tensor data
+  //@@     from the inference result for a response.
+  //@@     For example, this duration should include the time to copy output
+  //@@     tensor data from the GPU.
+  //@@
+  StatisticDuration compute_output = 2;
+
+  //@@  .. cpp:var:: StatisticDuration success
+  //@@
+  //@@     Cumulative count and duration for successful inference
+  //@@     responses.
+  //@@
+  StatisticDuration success = 3;
+
+  //@@  .. cpp:var:: StatisticDuration fail
+  //@@
+  //@@     Cumulative count and duration for failed inference
+  //@@     responses.
+  //@@
+  StatisticDuration fail = 4;
+}
+
+//@@
+//@@.. cpp:var:: message InferResponseStatistics
+//@@
+//@@   Inference response statistics.
+//@@
+message InferResponseStatistics
+{
+  //@@
+  //@@.. cpp:var:: message InferResponseStat (repeated)
+  //@@
+  //@@   Cumulative statistics corresponding to responses generated by the
+  //@@   requests that create certain number of responses.
```

Comment:
This is a bit confusing to read. Are these cumulative for a single request with multiple responses? Or cumulative across all responses?

Reply:
Yes. It is rewritten.

Reply:
It is cumulative across multiple responses for a request.